src/Microsoft.DotNet.Build.Tasks.Packaging/src/GenerateRuntimeGraph.cs

```diff
@@ -33,88 +35,81 @@ public class GenerateRuntimeGraph : PackagingTask
         ///   AdditionalQualifiers: A list of strings delimited by semi-colons that represent the additional qualifiers for this RID.
         ///     Additional qualifers do not stack, each only applies to the qualifier-less RIDs (so as not to cause combinatorial 
         ///     exponential growth of RIDs).
+        ///
+        ///   *Use at your own risk*
```

Comment:
I'm not sure I understand this warning.  Can it be explained further?

Reply:
Omitting RIDs from the graph can breaks the compatibility precedence we ensure when generating RIDs/imports from RuntimeGroups.  This makes it easy to create a RID fallback chain that doesn't match the rest of the RIDs and therefore is hard for developers/package authors to reason about.  Only use these options for cases where you know what you are doing and have carefully reviewed the resulting RID fallbacks.

Reply:
Sounds like a perfect comment to put into this code 😉

Reply:
I'll add a comment to the source.